pkgs/applications/office/libreoffice/default-gen-shell.nix

```diff
@@ -19,9 +19,11 @@ stdenv.mkDerivation {
     builder = ./download-list-builder.sh;
   };
 
+  buildInputs = [ python36 ];
```

Comment:
Why use a pre-release of Python 3.6? Please use `python3` which at this time refers to `python35`.